planex/cmd/pin.py

```diff
@@ -147,6 +147,10 @@ def parse_args_or_exit(argv=None):
     parser.add_argument("--patchqueue", default="master",
                         help="Value for the patchqueue field of the pin file. "
                              "Defaults to master. ")
+    parser.add_argument("--to", default=None,
```

Comment:
Could you please call this `-o`/`--output`?   I think users are more likely to expect that than `--to`.

Reply:
I think `None` is the default anyway.

Reply:
@euanh `-o` is good. I was wondering if it would be better to preserve the old behaviour instead of having to pass the output file.

@srowe true but I tend to be explicit to avoid breaking things if I move the flags to a an argument. I'd keep the explicit default if you don't mind